README.md

```diff
@@ -1,22 +1,48 @@
 # Woodpecker
 
-A fork of the Drone CI system.
-
-- Based on the v0.8 code tree
-- Fully Apache 2.0, no tiers
+Woodpecker is a Community fork of the Drone CI system.
 
 [![Go Report Card](https://goreportcard.com/badge/github.com/woodpecker-ci/woodpecker)](https://goreportcard.com/report/github.com/woodpecker-ci/woodpecker) [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) [![https://discord.gg/fcMQqSMXJy](https://img.shields.io/discord/838698813463724034.svg)](https://discord.gg/fcMQqSMXJy)
 
 ![woodpecker](docs/docs/images/woodpecker.png)
 
-## .drone.yml
+# Contribution
+
+## Maintainers
+
+To make sure every PR is checked, we have **team maintainers**.
+Every PR **MUST** be reviewed by at least **one** maintainer (or owner) before it can get merged.
+A maintainer should be a contributor and contributed at least 4 accepted PRs.
+A contributor should apply as a maintainer in the [Discord](https://discord.gg/fcMQqSMXJy) #develop channel.
+The owners or the team maintainers may invite the contributor.
+A maintainer should spend some time on code reviews.
+If a maintainer has no time to do that, they should apply to leave the maintainers team and
+we will give them the honor of being a member of the [advisors
+team](https://github.com/orgs/woodpecker-ci/teams/advisors/members).
+Of course, if an advisor has time to code review, we will gladly welcome them back to the maintainers team.
+If a maintainer is inactive for more than 3 months and forgets to leave the maintainers team,
+the owners may move him or her from the maintainers team to the advisors team.
+For security reasons, Maintainers should use 2FA for their accounts and if possible provide gpg signed commits.
+https://help.github.com/articles/securing-your-account-with-two-factor-authentication-2fa/
+https://help.github.com/articles/signing-commits-with-gpg/
+
+## Owners
+
+Since Woodpecker is a pure community organization without any company support, to keep the development healthy we will elect two owners every year.
+All maintainers may vote to elect up to two candidates. When the new owners have been elected, the old owners will give up ownership to the newly elected owners.
+If an owner is unable to do so, the other owners will assist in ceding ownership to the newly elected owners.
+For security reasons. Owners must use 2FA. https://help.github.com/articles/securing-your-account-with-two-factor-authentication-2fa/
```

Comment:
Wouldn't it be nice to place this part into a `CONTRIBUTING.md` file and link to it from the readme?